testsuite/tests/typing-layouts-bits16/alloc.ml

```diff
@@ -1,7 +1,7 @@
 (* TEST
- include stdlib_beta;
+ include stdlib_stable;
  flambda2;
- flags = "-extension small_numbers_beta -extension layouts_beta";
+ flags = "-extension small_numbers -extension layouts_beta";
```

Comment:
Is extension `layouts_beta` needed?  I think it should not be.

(And similar comment for all the other unboxed small numbers tests).